src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/ScriptActionsOnRunningCluster.cs

```diff
@@ -122,7 +122,7 @@ public void TestScriptActionsOnRunningCluster()
                     var result = client.Clusters.ExecuteScriptActions(resourceGroup, dnsName, executeScriptActionParams);
                     Assert.Equal(result.StatusCode, HttpStatusCode.OK);
                     Assert.Equal(result.State, AsyncOperationState.Failed);
-                    Assert.Equal(result.ErrorInfo.Message, "ScriptExecutionFailed");
```

Comment:
We have released Spark 2.0 to production but without SDK support customers cannot automate their cluster creation workflow. Please let me know if you have more comments or signoff if the changes look good.